src/web/releases.rs

```diff
@@ -178,13 +178,19 @@ async fn get_search_results(pool: Pool, query_params: &str) -> Result<SearchResu
         }
     });
 
-    let releases: CratesIoSearchResult = HTTP_CLIENT
-        .get(url)
-        .send()
-        .await?
-        .error_for_status()?
-        .json()
-        .await?;
+    let releases: CratesIoSearchResult = retry_async(
+        || async {
+            Ok(HTTP_CLIENT
+                .get(url.clone())
+                .send()
+                .await?
+                .error_for_status()?)
+        },
+        3,
```

Comment:
I'm not sure if we would really want to wait up to 14 seconds for this, maybe just have 1 retry attempt?

Reply:
In my mind I would prefer to wait that long for the result. 

Since it's all async, it won't hurt us much.

Reply:
The problem is latency - if we take that long to return a response, I think the client will give up on this request. I'd rather retry after 5 seconds or something like that.

Reply:
I don't fully follow, 

what behaviour are you proposing here?